caffe2/python/serialized_test/generate_serialized_tests.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import os
+import shutil
+
+import caffe2.python.serialized_test.collect_tests as collect
+from typing import Text
+import pickle
```

Comment:
Please don't use pickle. It's not safe, and if we update python classes, we may not be able to load the test data.
Please serialize the operator through protobuf, and data serialized using .npz format.

Here is some example: 
https://github.com/pytorch/pytorch/blob/master/test/onnx/test_operators.py#L74